tests/client.py

```diff
@@ -283,11 +287,12 @@ def get_companies(self, since=''):
 
         return records
 
-    def get_contact_lists(self, since='', list_id=''):
+    def get_contact_lists(self, since='', list_id='', pagination=False):
         """
         Get all contact_lists by paginating using 'has-more' and 'offset'.
         """
         url = f"{BASE_URL}/contacts/v1/lists"
+        page_size = self.BaseTest.expected_metadata()['contact_lists'].get(self.BaseTest.EXPECTED_PAGE_SIZE)
```

Comment:
same as above, we should avoid using dict keys directly in favor of get() with default fallback if there is more than one get()